app/models/game.rb

```diff
@@ -2,6 +2,8 @@ class Game < ActiveRecord::Base
   has_many :pieces
   delegate :pawns, :knights, :bishops, :rooks, :queens, :kings, to: :pieces
 
+  attr_accessor :winner_id
```

Comment:
Fixed the black/white player to be an id/integer. Easy fix, thanks for that. Still getting the same error.

Also, :winner_id is a field in the db. I added the attr_accessor as an attempt to hack the bug away. Didn't work.

I'm guessing that the button isn't calling the right method or the routes are messed up. This is confusing for me. Still couldn't figure it out after a couple hours of research. If someone else wants to try to pick up this task from here, feel free.

If I can see how this is done, I can probably finish off the castling as well.